test-utils/src/lib.rs

```diff
@@ -23,6 +23,7 @@ fn run(command: &str, args: Vec<&str>) -> R<()> {
     }
 }
 
+#[derive(Debug)]
```

Comment:
i don't think you make use of this in this PR

Reply:
Yes, removed.